### PR TITLE
update dev docs

### PR DESCRIPTION
**Pull request by oliver-sanders**
Small update to dev docs.

### CODE REVIEW

File: docs/howto/cylc-ui.md [hunk 1]
```diff
@@ -1,60 +1,113 @@
...
+## Development Installation
+
+> **Note**: We recommend developing in a Conda environment as this provides a
+  more standard setup though if you ensure the non-python dependencies
+  are installed correctly you can bypass this.
+
 1. Create a clean conda install:
   ```bash
-  conda create -n my-example-env python=3.7 --yes
+  conda create -n my-example-env python=3.7 configurable-http-proxy --yes
```
Comment (oliver-sanders) [inline docs/howto/cylc-ui.md:11]: Advise installing `configurable-http-proxy` via Conda.

No benefit to installing via yarn as we don't provide version pinning.

File: docs/howto/cylc-ui.md [hunk 2]
```diff
@@ -1,60 +1,113 @@
...
-  ```
-  where `${DEVDIR}/cylc-ui` is the location
-  of your local (git) worktree of the project.
-  
+
+   > **Note:** We prefer `yarn`.
+
+   ```bash
+   cd $DEVDIR/cylc-ui
+   yarn install
```
Comment (oliver-sanders) [inline docs/howto/cylc-ui.md:44]: npm => yarn

File: docs/howto/cylc-ui.md [hunk 3]
```diff
@@ -1,60 +1,113 @@
...
 
+   Fire up jupyterhub as before, open a browser and navigate to
+   `http://localhost:8000/`.
 
+5. Stop it:
 
+   (otherwise it will just keep running forever)
 
-*It is important that jupyterhub is not installed on conda after nodejs, or it
-will fatally downgrade nodejs.
```
Comment (oliver-sanders) [inline docs/howto/cylc-ui.md:60]: Looks like this is no longer the case - https://github.com/conda-forge/jupyterhub-feedstock/blob/master/recipe/meta.yaml#L81

Comment (kinow) [reply docs/howto/cylc-ui.md:60]: :+1: wasn't aware of that. Good to know!

**Review by kinow — APPROVED** [review]

File: docs/howto/cylc-ui.md [hunk 4]
```diff
@@ -1,60 +1,113 @@
...
-  ```bash
-  cd $DEVDIR/cylc-ui
-  npm install --prefix="${DEVDIR}/cylc-ui"
-  npm install -g configurable-http-proxy
-  ```
-  where `${DEVDIR}/cylc-ui` is the location
-  of your local (git) worktree of the project.
-  
+
+   > **Note:** We prefer `yarn`.
```
Comment (wxtim) [inline docs/howto/cylc-ui.md:40]:
```suggestion
> **Note** In the past we have used `npm`, but now prefer `yarn`.
```

Comment (oliver-sanders) [reply docs/howto/cylc-ui.md:40]: Useful?

File: docs/howto/cylc-ui.md (hunk 2 above)
Comment (datamel) [inline docs/howto/cylc-ui.md:44]: Commenting on this as I have been through this recently rebuilding environment from scratch on MO computer. We don't get yarn as standard so, for completeness of instruction... perhaps a warning that might need to do a npm install yarn first?

Comment (oliver-sanders) [reply docs/howto/cylc-ui.md:48]: Done.

**Review by hjoliver — APPROVED** [review]

**Pull request merged**